tests/scripts/release-post.test.js

```diff
@@ -172,19 +171,6 @@ test('fetchChangelogBody(<version>)', (t) => {
     }, t.fail)
   })
 
-  t.test('ensures notable changes header are prefix with `###`', (t) => {
```

Comment:
If I remember well it's because notable changes are always prefixed now making the test redundant.